### PR TITLE
[Magiclysm] New freezeing touch spell

### PR DESCRIPTION
**Pull request by Ryan7251**
#### Summary
Content "This new spell is an upgraded version of the chilling touch spell it not only deals more damage but will also slow down your foes."

#### Purpose of change
I wanted to make an Empowered version of chilly touch.

#### Testing

I used the debug menu and made sure it had no crashes and the slow down effect worked.

### CODE REVIEW

File: data/mods/Magiclysm/Spells/kelvinist.json [hunk 1]
```diff
@@ -397,5 +397,46 @@
...
+    "type": "SPELL",
+    "name": "Freezing Touch",
+    "description": " this is a sub-spell of Freezing touch .",
+    "shape": "blast",
+    "effect":"attack",
+    "effect_str": "slow_freeze_effect",
+    "valid_targets": [ "hostile" ],
+    "flags": [ "SILENT" ],
+    "min_duration":1000,
+    "max_duration":1000,
```
Comment (casswedson) [inline data/mods/Magiclysm/Spells/kelvinist.json:412]:
```suggestion
    "effect": "attack",
    "effect_str": "slow_freeze_effect",
    "valid_targets": [ "hostile" ],
    "flags": [ "SILENT" ],
    "min_duration": 1000,
    "max_duration": 1000,
```

File: data/mods/Magiclysm/Spells/kelvinist.json [hunk 2]
```diff
@@ -397,5 +397,46 @@
     "energy_source": "MANA",
     "difficulty": 5,
     "damage_type": "heat"
+  },
+  {
+    "id": "freezing_touch_slow_effect",
+    "type": "SPELL",
+    "name": "Freezing Touch",
+    "description": " this is a sub-spell of Freezing touch .",
```
Comment (casswedson) [inline data/mods/Magiclysm/Spells/kelvinist.json:405]:
```suggestion
    "description": "This is a sub-spell of Freezing touch.",
```

File: data/mods/Magiclysm/Spells/kelvinist.json [hunk 3]
```diff
@@ -397,5 +397,46 @@
...
+  {
+    "id": "freezing_touch",
+    "type": "SPELL",
+    "name": "Freezing Touch",
+    "description": "Your hands freeze anything they touch at temperatures so cold it slows down your foes.",
+    "valid_targets": [ "hostile" ],
+    "flags": [ "SOMATIC", "NO_PROJECTILE" ],
+    "components": "spell_components_freezing_touch",
+    "effect": "attack",
+    "effect_str": "cold",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/kelvinist.json:425]: what's this?

Comment (Ryan7251) [reply data/mods/Magiclysm/Spells/kelvinist.json:425]: umm sorry, I don't know what you mean if you mean the components I was trying to make a spell that used items like you wanted. but I'm going to take a guess and say I did something wrong I'm not seeing.

Comment (KorGgenT) [reply data/mods/Magiclysm/Spells/kelvinist.json:425]: "effect_str": "cold"
is the line this comment is pointing at.

Comment (actual-nh) [reply data/mods/Magiclysm/Spells/kelvinist.json:425]: > "effect_str": "cold"
> is the line this comment is pointing at.

It's the same in chilling touch.

Comment (KorGgenT) [reply data/mods/Magiclysm/Spells/kelvinist.json:425]: ah yes, carry on.

**Comment by Ryan7251** [comment]
Welp seems there was an issue with a cpp file can someone tell me how to remove a pull request I have no idea how to work with cpp files so I will have to trash this pull sadly.

**Comment by KorGgenT** [comment]
what? you didn't change any cpp that i can see

**Comment by Ryan7251** [comment]
> what? you didn't change any cpp that i can see

I did not change anything i'm taking about just some issue with a cpp file if you look at files changed the last thing shown talks about a check or something.
tests/rng_test.cpp

**Comment by actual-nh** [comment]
@Ryan7251: Actually, no, don't worry. Since there is some randomness in the game, some of the tests run to check to see if the randomness is right - but this unfortunately means that bad luck can cause a test fault.

**Pull request closed without merging**